README.md

```diff
@@ -697,6 +697,7 @@ Table of Contents
   * [raidboxes.io](https://raidboxes.io/) — Managed WordPress hosting with lots of included features. Free for developers and agencies, comes with custom domain support.
   * [readthedocs.org](https://readthedocs.org/) — Free documentation hosting with versioning, PDF generation and more
   * [render.com](https://render.com) — A unified platform to build and run all your apps and web app free SSL, a global CDN, private networks and auto deploys from Git, free for static web page.
+  * [sturtz.cf](http://sturtz.cf) - Create a free website! For Developers!
```

Comment:
thank you, please add a few notes about the free tier - for example how much space, b/w and if its ad supported